buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "38.0.20200326160141"
+    const val VERSION = "38.0.20200327101347"
```

Comment:
@pocmo  could you use the latest ac version? `20200327130125` :)

Reply:
At the time of opening the PR this was the latest version. Can we leave the bumping to MickeyMoz? :)

Reply:
Sure, let me re-open it :)

Reply:
It will open a new PR. No need to open it now as it will fail without this here landing. :)